finance-tracker/app/controllers/user_stocks_controller.rb

```diff
@@ -5,10 +5,10 @@ def create
       stock = Stock.new_lookup(params[:ticker])
       stock.save
     end
-    begin
+    if stock.save
```

Comment:
this save the stock the 2nd time.
1. Why you want save twice
2. Why you want to save if you didn't make any changes to the object?